x/project/auth.go

```diff
@@ -25,57 +39,53 @@ func GetPubKeyGetter(keeper Keeper, didKeeper did.Keeper) ixo.PubKeyGetter {
 			signerDid := msg.GetSignerDid()
 			signerDoc, _ := didKeeper.GetDidDoc(ctx, signerDid)
 			if signerDoc == nil {
-				return pubKey, sdk.ErrUnauthorized("signer did not found").Result()
+				return pubKey, sdkerrors.Wrap(sdkerrors.ErrUnauthorized, "signer did not found")
 			}
 			copy(pubKeyEd25519[:], base58.Decode(signerDoc.GetPubKey()))
 		default:
 			// For the remaining messages, the project is the signer
 			projectDoc, err := keeper.GetProjectDoc(ctx, msg.GetSignerDid())
 			if err != nil {
-				return pubKey, sdk.ErrInternal("project did not found").Result()
+				return pubKey, sdkerrors.Wrap(sdkerrors.ErrUnauthorized, "project did not found")
```

Comment:
Maybe we should use `ErrInvalidDid` instead?

Reply:
Correct